gravityflow/gravityflowincomingwebhook
# Fix "Add New" button appearing twice with Gravity Forms 2.5

## Description
The "Add New" button is displaying twice on the Incoming Webhooks form settings tab. For Gravity Forms 2.5, remove the one next to the title.

## Testing instructions
Check "Add New' appears only once for both Gravity Forms 2.4 and Gravity Forms 2.5.

## Automated Test Enhancements
N/A
 
## Screenshots
N/A

## Documentation Changes?
N/A

## Checklist:
